config/routes.rb

```diff
@@ -1,22 +1,14 @@
 Webapp::Application.routes.draw do
 
-  resources :home do
+  resources :assignments, only: [] do
     collection do
-      get :welcome
-    end
-  end
-
-  resources :assignments do
-    collection do
-      get :index
       get :knight
     end
   end
 
-  resources :regions do
+  resources :regions, except: [:show] do
     get 'recipients', :on => :member
     get 'request_rescue', :on => :member
-    as_routes
```

Comment:
@cllns I'm pretty sure those routes are unused.  You can verify similarly by running 

`bundle exec rake routes | grep regions`

to see which routes `as_routes` is adding.  They look unused to me -couldn't find views to support the actions or any links to those routes.

Related: ActiveScaffold looks like junk - would be good to remove eventually.